ext/TrixiNLsolveExt.jl

```diff
@@ -74,7 +73,7 @@ end
 # For details, see Proposition 3.2, Equation (3.3) from 
 # Hairer, Wanner: Solving Ordinary Differential Equations 2
 function Trixi.solve_a_butcher_coeffs_unknown!(a_unknown, num_stages, monomial_coeffs,
-                                               c_s2, c;
+                                               c;
```

Comment:
Is it possible to squeeze `c;` in the line above? If not, I would vote to put `verbose, max_iter = 100000` into this line as well.

Reply:
`c` cannot be put on the line above. I'll put `verbose, max_iter` in the same line as `c` then.